### PR TITLE
Added control scheme for 2d scenes

### PR DESCRIPTION
**Pull request by MaikoVDV**
Added 'orthographic' option to SpectatorSettings. Disables mouse looking & capturing. Makes W and S control y-axis movement.
This should significantly improve the experience in 2D scenes.
Added a 2D example scene.

### CODE REVIEW

**Comment by Shatur** [comment]
Could you update it to Bevy 0.11?

**Comment by MaikoVDV** [comment]
Done!
Also, I ran into an issue in the 3D example scene where PbrBundles wouldn't display materials (they were all that beautiful pinkish purple). Might this be a problem on my end or are you experiencing something similar?

**Comment by Shatur** [comment]
> Also, I ran into an issue in the 3D example scene where PbrBundles wouldn't display materials (they were all that beautiful pinkish purple). 

Thanks, fixed it in the latest master.

File: src/lib.rs [hunk 1]
```diff
@@ -163,7 +166,7 @@ fn spectator_update(
                 settings.base_speed
             };
 
-            let delta_axial = (forward - backward) * speed;
+            let delta_axial = if settings.orthographic { 0f32 } else { (forward - backward) * speed };
```
Comment (Shatur) [inline src/lib.rs]:
```suggestion
            let delta_axial = if settings.orthographic { 0.0 } else { (forward - backward) * speed };
```

**Review by Shatur — CHANGES_REQUESTED** [review]
I'm fine with this change, minor suggestion.

**Review by JonahPlusPlus — APPROVED** [review]

**Pull request merged**